src/components/PlayersArea.js

```diff
@@ -1,11 +1,36 @@
 /* flow */
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { playersCountDecrement, updatePlayersList } from '../actions';
+import PlayersList from './PlayersList';
 
 
 function PlayersArea() {
+  const totalPlayers = useSelector(state => state.app.teamSettings.totalPlayers);
+  const playersCount = useSelector(state => state.players.playersCount);
+  const playersList = useSelector(state => state.players.playersList);
+
+  const dispatch = useDispatch();
+
+  const updatePlayer = (e) => {
+    const playerId = parseInt(e.target.id);
+    // NOTE: decrementing count and updating state is semi-redundant and could be combined into one call.
+    dispatch(playersCountDecrement());
+    dispatch(updatePlayersList(playerId));
+  };
+
   return (
     <div>
-      Players
+      <p>Players: {playersCount}</p>
+      {playersCount > 0 && playersList ?
+        <PlayersList 
+          players={playersList}
+          handlePlayerOnClick={updatePlayer}
+        />
+        : totalPlayers > 0 ? 'No players remaining.'
```

Comment:
nested ternaries is discouraged, this isn't too hard to follow tho

Reply:
I didn't like this either. Would a better way to write this be to return early in the function body above the return, then a normal ternary in the return's body?

```
function PlayersArea() {
  const totalPlayers = useSelector(state => state.app.teamSettings.totalPlayers;
  ...
  // initial state is set to 0. Return earlier here
  if(totalPlayers == 0 && playersCount == 0) {
    return 'Use Configurations to set the number of players.'
  }

  return (
    <div>
    ...
    // keep the ternary here?
    {playersCount > 0 && playersList ?
      <PlayersList />
      : 'No players remaining.'
    }

    </div>
  )
}
```